phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/breezeDark/main.less

```diff
@@ -39,99 +45,99 @@
 }
 
 .button:hover {
-    -fx-border-color: #3daee9;
+    -fx-border-color: @focus-color;
 }
 
 .combo-box {
-    -fx-color: #bdc3c7;
+    -fx-color: @inactive-text-color;
     -fx-background-color: transparent;
-    -fx-border-color: #bdc3c7;
+    -fx-border-color: @inactive-text-color;
 }
 
 .combo-box:hover {
-    -fx-border-color: #3daee9;
+    -fx-border-color: @focus-color;
 }
 
 .combo-box:hover .arrow-button {
-    -fx-color: #3daee9;
+    -fx-color: @focus-color;
 }
 
 .combo-box-popup .list-view {
-    -fx-background-color: #31363b, #31363b;
+    -fx-background-color: @main-color, @main-color;
 }
 
 .combo-box .list-cell {
-    -fx-background: #31363b;
-    -fx-background-color: #31363b;
+    -fx-background: @main-color;
+    -fx-background-color: @main-color;
 }
 
 .combo-box-popup .list-view .list-cell:filled:hover {
-    -fx-background-color: #3daee9;
+    -fx-background-color: @focus-color;
 }
 
 .check-box .box {
     -fx-background-color: transparent;
-    -fx-border-color: #bdc3c7;
+    -fx-border-color: @inactive-text-color;
 }
 
 .check-box:hover .box {
     -fx-background-color: transparent;
-    -fx-border-color: #3daee9;
+    -fx-border-color: @focus-color;
 }
 
 .check-box:selected .mark {
-    -fx-color: #3daee9;
-    -fx-background-color: #3daee9;
+    -fx-color: @focus-color;
+    -fx-background-color: @focus-color;
 }
 
 .tab-header-area .tab-header-background {
-    -fx-background-color: #31363b;
+    -fx-background-color: @main-color;
 }
 
 .tab-header-area .tab:selected {
-    -fx-background-color: #3daee9;
+    -fx-background-color: @focus-color;
     -fx-focus-color: transparent;
     -fx-faint-focus-color: transparent;
 }
 
 .tab-header-area .tab {
     -fx-background-radius: 0;
-    -fx-background-color: #232629;
+    -fx-background-color: @background-color;
     -fx-border-width: 0;
     -fx-background-opacity: 0;
 }
 
 .list-view {
-    -fx-background-color: #232629;
-    -fx-border-color: #bdc3c7;
+    -fx-background-color: @background-color;
+    -fx-border-color: @inactive-text-color;
 }
 
 .list-cell {
-    -fx-background-color: #31363b;
+    -fx-background-color: @main-color;
     -fx-text-fill: -fx-text-base-color;
 }
 
 .list-cell:even {
-    -fx-background-color: #232629;
+    -fx-background-color: @background-color;
     -fx-text-fill: -fx-text-base-color;
 }
 
 .list-cell:filled:hover, .list-cell:filled:selected:focused, .list-cell:filled:selected {
-    -fx-background-color: #3daee9;
+    -fx-background-color: @focus-color;
     -fx-text-fill: -fx-text-base-color;
 }
 
 .text-field {
     -fx-border-radius: 0;
     -fx-background-radius: 0;
-    -fx-background-color: #31363b;
+    -fx-background-color: @main-color;
     -fx-border-color: transparent;
     -fx-text-fill: -fx-text-base-color;
```

Comment:
`-fx-text-base-color` in `-fx-text-fill: -fx-text-base-color;`.
In addition there are a lot of similar occurences in the less files.